Lib/fontmake/font_project.py

```diff
@@ -25,19 +25,12 @@
 from cu2qu.rf import fonts_to_quadratic
 from fontTools import subset
 from glyphs2ufo.glyphslib import build_masters, build_instances
+from mutatorMath.ufo import build as build_designspace
+from robofab.world import OpenFont
```

Comment:
sorry I just noticed the commit message:

> For the time being cu2qu only works with robofab, so don't try using defcon.

well, we shall fix this in cu2qu then. Once the pens are merged in cu2qu, defcon should work.